### ISSUE TITLE
Translation incomplete

### ISSUE THREAD

**Issue by lojiego** [Loopring/loopring-wallet-feedback#6]
This is likely a known issue, just putting it here for tracking .. 

under "settings" ... Daily Quota is described as, "Not as no limit" .. just a bad translation. Should be "Has no limit"
Also :   User Feedback page is still in Chinese.   needs to finish being translated.

**Comment by mfinestone** [comment]
Hey lojiego. I think I know who you are from our lengthy Discord chat today. Just want to say thanks again for the interest and for flagging these issues. Indeed, many of the issues, including translation imperfections are known. We will get to these within next few days. Cheers!

**Comment by lojiego** [comment]
Thanks Matt,

    Good to know .... the big issue I'm having is that the address that was
generated won't work. . .

The reason I got the LRC wallet was to store the LRC I was awarded at
Publish0x. (they just announced today they are stopping LRC and moved to a
different crypto ... if everyone doesn't withdraw by Dec 7th, they LRC gets
converted) ....  so I was going to post a step-by-step guide for installing
the  LRCWallet App .. seemed timely

However I get an error that the LRC Wallet address isn't valid when adding
it to Publish0x withdrawal.

Is that because it expects a L1 address, and this is only a L2?  not sure
why it states it's not valid ... but that's a pretty big issue for me at
the moment.

It is the correct 42 characters, but I can't use it to receive my LRC
tokens.

  - L

ps.  I updated the app this morning. The whitespace bug has been fixed,
nice work!   fyi, it took about :40-:50 min to download in-app, which was
very very slow. the download (same 26MB size) yesterday from the website
was done in a min or two
